de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/metadata/MetadataViewDefaultAppearance.java

```diff
@@ -254,4 +255,23 @@ public ImageResource saveToFileIcon() {
     public String saveMetadataToFile() {
         return toolbarMessages.saveMetadataMenuItem();
     }
+    
+    public String loading() {
+        return iplantDisplayStrings.loadingMask();
+    }
+
+    @Override
+    public String backgroudStyle() {
+        return "background:#fff;";
```

Comment:
I didn't realize you could do this!  Not sure if it's simpler this way or if it'd be better to put in a GSS file?